_docs/setup/kubernetes/ansible-install.md

```diff
@@ -57,8 +57,7 @@ The full list of configurable parameters is as follows:
 | `istio.dest` | The directory of the target machine where Istio will be installed | `~/.istio` (default) |
 | `istio.auth` | Boolean value to install Istio using MUTUAL_TLS | `true` and `false` (default) |
 | `istio.namespace` | The namespace where Istio will be installed | `istio-system` (default) |
-| `istio.addon` | Which Istio addons should be installed as well | This field is an array field, which by default contains `grafana`, `prometheus`, `zipkin` and `servicegraph` |
-| `istio.jaeger` | Whether or not Jaeger tracing should also be installed | `true` and `false` (default)|
+| `istio.addon` | Which Istio addons should be installed as well | This field is an array field, which by default contains `grafana`, `prometheus`, `zipkin`, `jaeger` (disables Zipkin if selected) and `servicegraph` |
```

Comment:
oxford comma would help clarity here e.g. "), and" although not a big deal.  Over time as you clean up this documentation might be something to consider for the future.

Reply:
Fixed :)